sdk/src/main/java/com/bugsnag/android/DefaultHttpClient.java

```diff
@@ -65,15 +67,16 @@ private int makeRequest(String urlString,
                 conn.addRequestProperty(entry.getKey(), entry.getValue());
             }
 
-            OutputStream out = null;
+            JsonStream stream = null;
 
             try {
-                out = conn.getOutputStream();
-                JsonStream stream = new JsonStream(new OutputStreamWriter(out));
+                OutputStream out = conn.getOutputStream();
+                Charset charset = Charset.forName("UTF-8");
+                BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(out, charset));
+                stream = new JsonStream(writer);
                 streamable.toStream(stream);
-                stream.close();
             } finally {
-                IOUtils.closeQuietly(out);
+                IOUtils.closeQuietly(stream);
```

Comment:
Will closing the `JsonStream` close the `OutputStream` as well?  There's no explicit closing of the `OutputStream`.

Reply:
Yes - also answered in previous comments in a bit more depth.

Reply:
Specifically: https://github.com/bugsnag/bugsnag-android/blob/master/sdk/src/main/java/com/bugsnag/android/JsonWriter.java#L423

Reply:
Ok, is this verifiable or testable at all?  It seems like something that could change across implementations/versions.

Reply:
I've added a test to check that JsonWriter closes its underlying stream - beyond that we'd just be testing the behaviour of the Java standard library.